2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -10,7 +10,11 @@ set -o pipefail
 export MONGODB_DATADIR=/var/lib/mongodb/data
 export CONTAINER_PORT=27017
 # Configuration settings.
-export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
+if [[ ${MONGODB_NOPREALLOC:-true} == "true" ]]; then
+  export MONGODB_PREALLOC="false"
+else
+  export MONGODB_PREALLOC="true"
+fi
```

Comment:
@bparees Thanks.

Changed a way how the default is computed. `MONGODB_PREALLOC` is always preferred. If it is not set, default is get according to `MONGODB_NOPREALLOC`.